tests/test_continuous_events.py

```diff
@@ -0,0 +1,33 @@
+from datetime import timedelta
+
+from aw_client import ActivityWatchClient
+
+client = ActivityWatchClient("aw-watcher-afk", testing=True)
+print(client.get_buckets())
+
+events = client.get_events("aw-watcher-afk-testing_erb-laptop-ubuntu")
```

Comment:
This needs to be generalized, or even better made into an argument to the script